raaghu-elements/src/rds-slider/rds-slider.tsx

```
@@ -139,19 +130,27 @@
       return [min, max];
     });
   };
+  const percent1 = ((value - Number(props.leftLabel)) / (Number(props.rightLabel) - Number(props.leftLabel))) * 100;
```

ESLint

[error] Expected indentation of 4 spaces but found 2.

(indent)